interface/patient_file/summary/immunizations.php

```diff
@@ -310,12 +310,12 @@ function saveImmunizationObservationResults($id,$immunizationdata)
 <!-- supporting javascript code -->
 <script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-2-1/index.js"></script>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
-<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/dialog.js"></script>
-
+<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/public/assets/jquery-min-1-9-1/index.js"></script>
+<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/public/assets/jquery-ui-min/index.js"></script>
```

Comment:
I will follow your lead on this one because my experience is that removing versions of jquery has different effects. Some times the older newer version is backwards compatible and sometimes it is not. I have removed older version with disastrous results. Other times its is ok. I will remove the 1.2.1 and upgrade to the 1.10.4 and see what happens.